prow/cmd/hook/main.go

```diff
@@ -171,5 +183,21 @@ func main() {
 	// Serve plugin help information from /plugin-help.
 	http.Handle("/plugin-help", pluginhelp.NewHelpAgent(pluginAgent, githubClient))
 
+	var appClient config.Client
+	yaml.Unmarshal(bytes.TrimSpace(gitAppClientRaw), &appClient)
+	cookie := sessions.NewCookieStore(securecookie.GenerateRandomKey(64))
+
+	gitOAuthConfig := &configAgent.Config().GitOAuthConfig
+	gitOAuthConfig.InitGitOAuthConfig(&appClient, cookie)
+
+	goa := gitoauth.NewGitOAuthAgent(gitOAuthConfig, logrus.WithField("component", "gitoauth"))
```

Comment:
component is still `hook` - you probably want to use `client` instead.

Reply:
done